src/Moq/Mock.Generic.cs

```diff
@@ -859,6 +897,196 @@ public void VerifySet(Action<T> setterExpression, Func<Times> times, string fail
 			Mock.VerifySet(this, expression , times(), failMessage);
 		}
 
+		/// <summary>
+		///   Verifies that an event was added to the mock.
+		/// </summary>
+		/// <param name="addEventExpression">Expression to verify.</param>
+		/// <exception cref="MockException">The invocation was not performed on the mock.</exception>
+		/// <include file='Mock.Generic.xdoc' path='docs/doc[@for="Mock{T}.VerifyAdd(expression)"]/*'/>
+		public void VerifyAdd(Action<T> addEventExpression)
```

Comment:
Actually, I took the `SetupSet` and `VerifySet` as an example. Should we fix them also?

Reply:
> Should we fix them also?

No, because these public methods have already been out there for a long time. Changing their parameters' names now would be a breaking change (specifically, for anyone using named parameters: `mock.Verify(..., param: ..., ...)`). Let's avoid this.

Reply:
👍